src/index.d.tsx

```diff
@@ -7,3 +7,8 @@ declare module '*.css' {
     const content: { [className: string]: string };
     export default content;
 }
+
+declare module 'highlight-javascript-syntax' {
```

Comment:
I was super surprised to find that `npm install @types/highlight-javascript-syntax` wasn't found. 🤷‍♂️

Reply:
Yeah, not exactly a popular one. Just super simple

Reply:
I didn't know they had some popularity threshold on that. I thought Microsoft had some sort of bot that published `@types`.

Reply:
> I didn't know they had some popularity threshold on that. I thought Microsoft had some sort of bot that published @types.

Whoa really - I thought the entirety of definitely-typed was manually created?

Reply:
DefinitelyTyped is manual, but I thought `@types` covered more than just what's available on DT.